qiskit/visualization/pulse_v2/device_info.py

```diff
@@ -124,7 +124,11 @@ def create_from_backend(cls, backend: BaseBackend):
         for qind, u_lo_mappers in enumerate(configuration.u_channel_lo):
             temp_val = .0 + .0j
             for u_lo_mapper in u_lo_mappers:
-                temp_val += defaults.qubit_freq_est[u_lo_mapper.q] * complex(*u_lo_mapper.scale)
+                if isinstance(u_lo_mapper.scale, complex):
+                    scale = u_lo_mapper.scale
+                else:
+                    scale = complex(*u_lo_mapper.scale)
```

Comment:
This should already be a complex, why is it not? I think this may be related to #5410?

Reply:
because of old mock devices...

Reply:
oh I found the solution. I think I can update this. https://github.com/Qiskit/qiskit-terra/issues/5426